builds.sh

```diff
@@ -56,6 +69,10 @@ _resolve() {
 	' || exit 1
 }
 
+declare -A imageArchResolved=(
+	#["image"]="arch"
```

Comment:
This should make the format more clear, something like:
```suggestion
	#["image"]="{...}" # JSON result of bashbrew remote arches
```

Reply:
In that case, should the cached "null" value be `{}`?

Reply:
I think `null` is fine if we're going to _pre_-test against it -- if we want to use a different value, it should be something like `{"arches":{}}` that we wouldn't _have_ to pre-test against (ie, that the `jq` would just "do the right thing" with), but IMO `null` is more clear as a sentinel that the lookup had some kind of problem (such as the image not existing).